### PR TITLE
Try upwinding with smaller dt

### PR DESCRIPTION
**Pull request by charleskawczynski**
Per [this request](https://github.com/CliMA/ClimaAtmos.jl/issues/409#issuecomment-1120230938).

### CODE REVIEW

File: .buildkite/longruns/pipeline.yml [hunk 1]
```diff
@@ -50,3 +50,12 @@ steps:
         agents:
           slurm_cpus_per_task: 8
 
+  - group: "Experimental Long runs"
+
+    steps:
+
+      - label: ":computer: held suarez (ρe)"
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe --forcing held_suarez --dt 300 --t_end 69120000 --fps 30 --job_id longrun_hs_rhoe_dt200 --dt_save_to_sol 21600" # 800 days
```
Comment (szy21) [inline .buildkite/longruns/pipeline.yml:58]:
```suggestion
        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe --forcing held_suarez --dt 300 --t_end 69120000 --fps 30 --job_id longrun_hs_rhoe_dt300 --dt_save_to_sol 21600" # 800 days
```

File: .buildkite/longruns/pipeline.yml [hunk 2]
```diff
@@ -50,3 +50,12 @@ steps:
...
+        artifact_paths: "longrun_hs_rhoe_dt200/*"
```
Comment (szy21) [inline .buildkite/longruns/pipeline.yml:59]:
```suggestion
        artifact_paths: "longrun_hs_rhoe_dt300/*"
```

**Review by szy21 — APPROVED** [review]

File: .buildkite/longruns/pipeline.yml (hunk 1 above)
Comment (charleskawczynski) [reply .buildkite/longruns/pipeline.yml:58]: 🤦🏻 thank you!

**Pull request merged**